src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1654,8 +1654,37 @@ public void cannotCreateRequestWhenRequesterHasActiveRequestManualBlocks() {
     assertThat(postResponse, hasStatus(HTTP_UNPROCESSABLE_ENTITY));
     assertThat(postResponse.getJson(), hasErrors(1));
     assertThat(postResponse.getJson(), hasErrorWith(allOf(
-      hasMessage("Patron blocked from requesting"))));
-      hasParameter("reason", "Display description");
+      hasMessage("Patron blocked from requesting"),
+      hasParameter("reason", "Display description")))
+    );
+  }
+
+  @Test
+  public void cannotCreateRequestWhenRequesterHasActiveRequestManualBlocksAndExpirationDateIsNotSet() {
+    final IndividualResource item = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requester = usersFixture.rebecca();
+    final UUID pickupServicePointId = servicePointsFixture.cd1().getId();
+
+    final UserManualBlockBuilder userManualBlockBuilder = getManualBlockBuilder()
+      .withRequests(true)
+      .withUserId(String.valueOf(requester.getId()));
```

Comment:
Can a comment be added here to highlight that the manual block's expiration date is intentionally not been set?